static/js/utils.ts

```diff
@@ -0,0 +1,7 @@
+
+export function isLoggedIn() {
+    if (document.body.dataset["loggedIn"]) {
+        return parseInt(document.body.dataset["loggedIn"])
+    }
+    return false;
```

Comment:
The return type of this function ends up being `number | boolean`. That's not a huge problem because both can be tested for truthiness, but you probably intend to just be returning a `boolean`. Test the data attribute by doing:

```
document.body.dataset.loggedIn === '1'
```